src/data/fullness.txt

```diff
@@ -784,7 +784,7 @@ ridiculous sandwich	3	1	awesome	11	10-12	10-12	10-12
 roasted duck	2	4	decent	3-4	0	7-12	0
 roasted marshmallow	1	1	crappy	1	0	0	0	3 Mallowed Out (+all stats)
 roasted vegetable focaccia	2	1	EPIC	15-17	40-80	40-80	0	50 Feeling Fancy (fam exp +10, +15 Muscle, +200% Mysticality)
```

Comment:
the calzone adventures were nerfed, but the focaccia adventures were not?

Reply:
seems like it. the calzone was simultaneously turned into a pizza and nerfed around cbb's release, guessing focaccia must've been changed at some later point